### ISSUE TITLE
Indexing over set<text> not working

### ISSUE THREAD

**Issue by dioptre** [strapdata/elassandra#157]
Using release:
https://github.com/strapdata/elassandra/releases/tag/v5.5.0.9

No plugins

OpenJDK 1.8

Debian Stretch

Expecting 
```curl -XGET 'http://localhost:9200/hd/users/_search?q=test&pretty=true'```

To return a result for:
set < text > with values {'test2', 'test'}

With schema like:
```
create table users (
  username     text,
  categories set<text>,
  primary key       (username)
);
```
I've done an index rebuild and refresh with no effect.

_mapping gives:
```
        "properties" : {
          "categories" : {
            "type" : "keyword",
            "cql_collection" : "set"
          },
```

Thanks

**Comment by vroyer** [comment]
Hi Andrew,
It’s a regression of version 5.5 not including the fix #112.
I’m going to release a fixed version today.
Thanks for reporting this issue.

**Comment by dioptre** [comment]
Let me know if you release it, and I'll happily provide some feedback.

Thanks
A
